src/hooks/Tokens.ts

```diff
@@ -240,16 +240,19 @@ export const fetchTokenByAddress = async (address: string, chainId: ChainId) =>
 export const fetchListTokenByAddresses = async (address: string[], chainId: ChainId) => {
   const response = await axios.get(`${KS_SETTING_API}/v1/tokens?addresses=${address}&chainIds=${chainId}`)
   const tokens = response?.data?.data?.tokens ?? []
-  return tokens.map(formatAndCacheToken)
+  return filterTruthy(tokens.map(formatAndCacheToken)) as WrappedTokenInfo[]
 }
 
 export const formatAndCacheToken = (tokenResponse: TokenInfo) => {
   try {
     const tokenInfo = new WrappedTokenInfo(tokenResponse)
+    if (!tokenInfo.decimals) {
```

Comment:
what about decimal 0?

Reply:
0 => hide

Reply:
why? it should be valid

Reply:
updated. missing condition, nice catch 💯 
```
 if (!tokenInfo.decimals && !tokenInfo.symbol && !tokenInfo.name) {
      return
    }
```